docs/collections.md

```diff
@@ -1,107 +1,90 @@
 # 🍱 Collections
 
-## ❓ What are Collections?
+## What are Collections?
 
-[Fetching data is easy in Astro][docs-data]. But what if you wanted to make a paginated blog? What if you wanted an easy way to sort data, or filter data based on part of the URL? Or generate an RSS 2.0 feed? When you need something a little more powerful than simple data fetching, Astro’s Collections API may be what you need.
+Astro Collections help you break up a larger set of data into multiple pages. Examples of use-cases include:
 
-An Astro Collection is similar to the general concept of Collections in static site generators like Jekyll, Hugo, Eleventy, etc. It’s a general way to load an entire data set. But one big difference between Astro Collections and traditional static site generators is: **Astro lets you seamlessly blend remote API data and local files in a JAMstack-friendly way.** To see how, this guide will walk through a few examples. If you’d like, you can reference the [blog example project][example-blog] to see the finished code in context.
+- Pagination: `/posts/1`, `posts/2`, etc.
+- Grouping content by author: `/posts/fred`, `/posts/matthew`, etc.
+- Grouping content by some tag: `/tags/red`, `tags/blue`, etc.
+- Working with remote data
+- Mixing remote and local data
 
-## 🧑‍🎨 How to Use
+**When to use Collections: When you need to reuse a single template to generate multiple pages from a larger dataset.** If you just want to generate a single page (ex: a long list of every post on your site) then you can just fetch that data on a normal Astro page without using the Collection API.
 
-By default, any Astro component can fetch data from any API or local `*.md` files. But what if you had a blog you wanted to paginate? What if you wanted to generate dynamic URLs based on metadata (e.g. `/tag/:tag/`)? Or do both together? Astro Collections are a way to do all of that. It’s perfect for generating blog-like content, or scaffolding out dynamic URLs from your data.
 
-Let’s pretend we have some blog posts written already. This is our starting project structure:
+## Collections API
 
-```
-└── src/
-    └── pages/
-        └── post/
-            └── (blog content)
-```
+To create a new Astro Collection, you must do three things:
 
-The first step in adding some dynamic collections is deciding on a URL schema. For our example website, we’re aiming for the following URLs:
+```js
+// 1. Create a new file in the `src/pages` directory that starts 
```

Comment:
I think steps 1, 2, and 3 should live in plain text and not a code block. Step 1 actually doesn’t relate to code at all, and would be easier to read in a paragraph IMO.

Steps #2 and #3 do relate to code, but I think more context is needed to describe where these go in a file. Should these be a diff?

Reply:
SGTM